packages/vm/test/api/runBlock.spec.ts

```diff
@@ -27,7 +27,7 @@ import type { TypedTransaction } from '@ethereumjs/tx'
 const testData = require('./testdata/blockchain.json')
 const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Berlin })
 describe('runBlock tests', () => {
-  it('runBlock() -> successful API parameter usage', async () => {
+  describe('runBlock() -> successful API parameter usage', async () => {
```

Comment:
What would you say to just removing the nested `describe` anyway?  I don't think it gives us any advantage in this case.

Reply:
Yeah sure, just wanted to minimally change what was already there, but agree they don't do much, will remove.

Reply:
Removed the top-level one.